A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_Fuel.html

```diff
@@ -0,0 +1,164 @@
+            <circle cx="190" cy="210" r="15" />
+            <line x1="175" y1="210" x2="205" y2="210" />
+        </g>
+
+        <g id="leftPump1_Off">
+            <rect class="ValvePumpInactive" x="177.5" y="300" width="25" height="25" />
+            <line class="ValvePumpInactive" x1="182.5" y1="312.5" x2="197.5" y2="312.5" />
+        </g>
+        <g id="leftPump1_On">
+            <rect x="177.5" y="300" width="25" height="25" />
+            <line x1="190" y1="300" x2="190" y2="325" />
+        </g>
+
+        <g id="leftPump2_Off">
+            <rect class="ValvePumpInactive" x="207.5" y="300" width="25" height="25" />
+            <line class="ValvePumpInactive" x1="212.5" y1="312.5" x2="227.5" y2="312.5" />
+        </g>
+        <g id="leftPump2_On">
+            <rect x="207.5" y="300" width="25" height="25" />
+            <line x1="220" y1="300" x2="220" y2="325" />
+        </g>
+
+        <!--Right-->
+        <g id="rightPipes">
+            <line class="FlowShape" x1="410" y1="300" x2="410" y2="185" />
+            <line class="FlowShape" x1="410" y1="250" x2="325" y2="250" />
+            <line class="FlowShape" x1="325" y1="250" x2="325" y2="290" />
+            <line class="FlowShape" x1="410" y1="275" x2="380" y2="275" />
+            <line class="FlowShape" x1="380" y1="275" x2="380" y2="298" />
+            <line class="MainShape" x1="390" y1="160" x2="340" y2="150" />
+        </g>
+
+        <g id="rightValve_On">
+            <circle cx="410" cy="210" r="15" />
+            <line x1="410" y1="195" x2="410" y2="225" />
+        </g>
+        <g id="rightValve_Off">
+            <circle cx="410" cy="210" r="15" />
+            <line x1="395" y1="210" x2="425" y2="210" />
+        </g>
+
+        <g id="rightPump1_Off">
+            <rect class="ValvePumpInactive" x="367.5" y="300" width="25" height="25" />
+            <line class="ValvePumpInactive" x1="372.5" y1="312.5" x2="387.5" y2="312.5" />
+        </g>
+        <g id="rightPump1_On">
+            <rect x="367.5" y="300" width="25" height="25" />
+            <line x1="380" y1="300" x2="380" y2="325" />
+        </g>
+
+        <g id="rightPump2_Off">
+            <rect class="ValvePumpInactive" x="397.5" y="300" width="25" height="25" />
+            <line class="ValvePumpInactive" x1="402.5" y1="312.5" x2="417.5" y2="312.5" />
+        </g>
+        <g id="rightPump2_On">
+            <rect x="397.5" y="300" width="25" height="25" />
+            <line x1="410" y1="300" x2="410" y2="325" />
+        </g>
+
+        <!--Middle-->
+        <g id="middleValve_On">
+            <circle cx="300" cy="250" r="15" />
+            <line x1="285" y1="250" x2="315" y2="250" />
+        </g>
+
+        <g id="middleValve_Off">
+            <circle cx="300" cy="250" r="15" />
+            <line x1="300" y1="235" x2="300" y2="265" />
+        </g>
+
+        <g id="middlePump1_Off">
+            <rect x="262.5" y="290" width="25" height="25" />
```

Comment:
Hi @ThePilot916 

I think the missing ValvePumpInactive here makes the closed middle pump 1 appears in green instead orange

![image](https://user-images.githubusercontent.com/121706/91665514-6bd1fe00-eaf6-11ea-8676-41b6b475950f.png)